<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupServerView</class>
 <widget class="QWidget" name="SetupServerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>518</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Account Information</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Account Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="accountName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="whatsThis">
             <string>Indicate the IMAP server. If you want to connect to a non-standard port for a specific encryption scheme, you can add &quot;:port&quot; to indicate that. For example: &quot;imap.foo.com:144&quot;.</string>
            </property>
            <property name="text">
             <string>IMAP Server:</string>
            </property>
            <property name="buddy">
             <cstring>imapServer</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="imapServer"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="whatsThis">
             <string>The username.</string>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
            <property name="buddy">
             <cstring>userName</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="userName"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="whatsThis">
             <string>The password.</string>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="buddy">
             <cstring>password</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="checkOptions">
         <property name="title">
          <string>Mail Checking Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="enableMailCheckBox">
            <property name="text">
             <string>Enable &amp;interval mail checking</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_1">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Check mail interval:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KIntNumInput" name="checkInterval">
              <property name="value">
               <number>1</number>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="suffix">
               <string> </string>
              </property>
              <property name="prefix">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Filtering</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="managesieveCheck">
         <property name="text">
          <string>Server supports Sieve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="sameConfigCheck">
         <property name="text">
          <string>Reuse host and login configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Managesieve port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntSpinBox" name="sievePortSpin">
           <property name="maximum">
            <number>2000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Alternate URL:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="alternateURL"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>IMAP Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="subscriptionEnabled">
              <property name="text">
               <string>Enable server-side subscriptions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KPushButton" name="subscriptionButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Serverside Subscription...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="disconnectedModeEnabled">
            <property name="text">
             <string>Enable disconnected mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoExpungeCheck">
            <property name="text">
             <string>Automaticall&amp;y compact folders (expunges deleted messages)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Trash folder:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Akonadi::CollectionRequester" name="folderRequester">
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="identityBox">
            <property name="title">
             <string>Identity Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QCheckBox" name="useDefaultIdentityCheck">
               <property name="toolTip">
                <string>Use the default identity for this account</string>
               </property>
               <property name="text">
                <string>Use &amp;default identity</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="identityLayout">
               <item>
                <widget class="QLabel" name="identityLabel">
                 <property name="toolTip">
                  <string>Select the KMail identity used for this account</string>
                 </property>
                 <property name="text">
                  <string>Identity:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="safeImap">
         <property name="whatsThis">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'HandelGotDLig'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;SSL/TLS&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt; is safe IMAP over port 993;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;STARTTLS&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt; will operate on port 143 and switch to a secure connection directly after connecting;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-weight:600;&quot;&gt;None&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt; will connect to port 143 but not switch to a secure connection. This setting is not recommended.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Connection Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="KPushButton" name="testButton">
            <property name="text">
             <string>Auto Detect</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Encryption:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="noRadio">
                <property name="text">
                 <string>None</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string>safeImapGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="sslRadio">
                <property name="text">
                 <string>SSL/TLS</string>
                </property>
                <attribute name="buttonGroup">
                 <string>safeImapGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="tlsRadio">
                <property name="text">
                 <string>STARTTLS</string>
                </property>
                <attribute name="buttonGroup">
                 <string>safeImapGroup</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="portSpin">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65534</number>
              </property>
              <property name="value">
               <number>143</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Authentication:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="authenticationCombo">
              <item>
               <property name="text">
                <string>Clear text</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>LOGIN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PLAIN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CRAM-MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DIGEST-MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NTLM</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GSSAPI</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Anonymous</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="testInfo">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="testProgress">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>Akonadi::CollectionRequester</class>
   <extends>QFrame</extends>
   <header>Akonadi/CollectionRequester</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="safeImapGroup"/>
 </buttongroups>
</ui>
